server/src/domain/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod game;
+pub mod model;
+pub mod player;
```

Comment:
I guess in Rust to make a module you need to declare it as a `mod`. This `domain/mod.rs` file declares all the things in `domain` as modules.